changelogs/unreleased/6518-pip-config-docs.yml

```diff
@@ -3,8 +3,5 @@ issue-nr: 6518
 change-type: major
 destination-branches: [master]
 sections:
-  feature: "A project-wide pip configuration can be set through the project.yml pip section"
-  upgrade-note: "Please refer to the `migration guide <https://docs.inmanta.com/inmanta-service-orchestrator/7/reference/projectyml.html#migrate-to-project-wide-pip-config>`_ to move to a project-wide pip configuration"
-  deprecation-note:
-    - "Allowing the installation of pre-release versions for modules through the :class:`~inmanta.module.InstallMode` is now deprecated in favour of the project.yml ``pip.pre`` section"
-    -
+  feature: "A project-wide pip configuration can be set through the project.yml pip section."
+  upgrade-note: "All projects now require a pip config in the `project.yml`: please refer to the [migration guide](migrate_to_project_wide_pip_config) to move to a project-wide pip configuration."
```

Comment:
This is a different type of link than the one in the change entry we discovered yesterday. If it works I prefer this one. Have you tested that it does?

Reply:
It is actually used in the linked [change entry](https://github.com/inmanta/inmanta-lsm/blob/master/changelogs/1.8.0/566-allow-type-updates.yml) (last line)
I checked that the link there works: https://docs.inmanta.com/inmanta-service-orchestrator/latest/CHANGELOG.html#id142